configs/components/oifs/oifs.env.yaml

```diff
@@ -228,12 +229,12 @@ compiletime_environment_changes:
         - 'OIFS_NETCDFF_LIB="-L$NETCDFFROOT/lib -lnetcdff"'
         # compilers and compile switches
         - 'OIFS_FC=$FC'
-        - 'OIFS_FFLAGS="-r8 -fp-model precise -align array32byte -O1 ${computer.OpenMP_flag} -xCORE_AVX2 -g -traceback -convert big_endian -fpe0"'
+        - 'OIFS_FFLAGS="-r8 -fp-model precise -align array32byte -O3 ${computer.OpenMP_flag} -xCORE_AVX2 -g -traceback -convert big_endian -fpe0"'
         - 'OIFS_FFIXED=""'
         - 'OIFS_FCDEFS="BLAS LITTLE LINUX INTEGER_IS_INT"'
         - 'OIFS_LFLAGS="$OIFS_MPI_LIB ${computer.OpenMP_flag}"'
         - 'OIFS_CC=$CC'
-        - 'OIFS_CFLAGS="-fp-model precise -O1 -xCORE_AVX2 -g -traceback -qopt-report=0 -fpe0 ${computer.OpenMP_flag}"'
+        - 'OIFS_CFLAGS="-fp-model precise -O3 -xCORE_AVX2 -g -traceback -qopt-report=0 -fpe0 ${computer.OpenMP_flag}"'
         - 'OIFS_CCDEFS="LINUX LITTLE INTEGER_IS_INT _ABI64 BLAS ${computer.OpenMP_CCDEFS}"'
         - 'MAIN_LDFLAGS=-openmp'
```

Comment:
**Systems involved**: AWI-CM3, FOCI-OpenIFS, OpenIFS standalone (@JanStreffing, @seb-wahl, @joakimkjellsson) in `Juwels`
    - **Evaluation**: compilation flags changed
    - **Details**: Wasn't this resulting into crashed simulations on the past? (@JanStreffing, do you recall that? It might well be that my memory is failing)
    - **Action required**: reply from @JanStreffing

Reply:
I ran approximately 1400 years for evaluation with O3. The model crashed 3 times, all of which looked like hardware failing. I will make a short speed test comparing O1 and O3. Probably we can stick with O1

Reply:
I will test the speed difference. If it is small we can stick with O1

Reply:
Slowdown for one month is from 69 seconds to 76. I went back to O1. If anyone wants maximum performance they can change it for themselves.

Reply:
Then let's go with O1 and avoid Juwels if we want to be fast ;-) 
/J